x-pack/legacy/plugins/maps/public/layers/layer_wizard_registry.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
+
+type LayerWizard = {
+  description: string;
+  icon: string;
+  id: string;
+  isIndexingSource?: boolean;
+  order: number; // number to control display order in UI. Lower numbers display first
```

Comment:
not sure if this is necessary. These numbers will become a mess and sources will require knowledge of other sources as to what the numbering-scheme exactly is.

With the layer-wizard, we are no longer restricted to register a source in the actual source-module. I'd register them in the desired order in another module.

Reply:
Ordering is necessary, otherwise the wizard cards will show up in what ever order there source files are loaded. Longer term this will be replaced with hierarchy, but this is needed to preserve the current order of the cards.

Reply:
I have removed ordering and replace with registering all wizards in a single file so order is determined by registration order.